src/ngraph/serializer.cpp

```diff
@@ -1693,7 +1693,33 @@ shared_ptr<Node> JSONDeserializer::deserialize_node(json node_js)
             node = make_shared<op::HardSigmoid>(args[0], args[1], args[2]);
             break;
         }
-        case OP_TYPEID::Interpolate: { break;
+        case OP_TYPEID::Interpolate:
```

Comment:
No serialize changes; need to use the attribute visitor.

Reply:
Done